doc/pages/apps/initialization-and-preferences-forms.md

```diff
@@ -267,7 +267,7 @@ This sample form also makes use of [translation functions](:apps/translations.ht
       entries.push(['string', PORT, 'Port'])
 
       values[COUNTRY_VARIANT] = Nuvola.config.get(COUNTRY_VARIANT)
-      entries.push(['header', _('Amazon Cloud Player')])
+      entries.push(['header', _('Amazon Butt Player')])
```

Comment:
This isn't correct.

Reply:
Oh man, so sorry! I'm embarrassed; this wasn't me, it was a silly extension I have installed 😅

I'll push an amended patch shortly.